tests/unit/test_platform.cpp

```diff
@@ -226,6 +230,36 @@ OT_TOOL_WEAK otError otPlatEntropyGet(uint8_t *aOutput, uint16_t aOutputLength)
     return error;
 }
 
+#if (OPENTHREAD_CONFIG_CRYPTO_LIB == OPENTHREAD_CONFIG_CRYPTO_LIB_PSA) && defined(MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG)
+/**
```

Comment:
I don't think you want this comment/description in the `doxygen` documentation (e.g., on the `openthread.io` website), so I suggest not using the `/**` format. Perhaps we can move it inside the function itself?